clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -718,7 +718,7 @@ public void testJoinGroupRequestVersion0RebalanceTimeout() {
     @Test
     public void testOffsetFetchRequestBuilderToString() {
         String allTopicPartitionsString = OffsetFetchRequest.Builder.allTopicPartitions("someGroup").toString();
-        assertTrue(allTopicPartitionsString.contains("<ALL>"));
+        assertTrue(allTopicPartitionsString.contains("groupId='someGroup', topics=null"));
```

Comment:
Do you think this test case has much value? I'd be inclined to get rid of it.

Reply:
I think it's ok to maintain a test on the `toString()` behavior